emcc.py

```diff
@@ -2235,27 +2251,31 @@ def check_memory_setting(setting):
     if settings.GLOBAL_BASE != -1:
       exit_with_error("ASan does not support custom GLOBAL_BASE")
 
-    max_mem = settings.INITIAL_MEMORY
+    user_mem = settings.INITIAL_MEMORY
```

Comment:
Maybe add a comment,
```
# user_mem : memory usable/visible by the user program.
# shadow_size : memory used by asan for shadow memory.
# total_mem : the sum of the above. this is the size of the wasm memory.
```
(is that right?)

Reply:
done.